.github/workflows/ci.yml

```diff
@@ -1,50 +1,42 @@
-# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
-# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+name: Continuous integration
 
-name: Node.js CI
-
-on:
-    push:
-        branches: [master]
-    pull_request:
-        branches: [master]
+on: 
+  - push
```

Comment:
The correct syntax should be either :
`on: push`
or
`on: [push]`
or
```
on:
    push:
```
Hint: No job was run for this PR, which means it's not working

Reply:
My bad this **may** be correct, just found out your PR is across forks : Maybe you don't have activated GA on your fork. And anyway fork jobs are never taken into account in PR checks.

We always need both push and PR events : PR to be fork-compatible, push because you may want to work without making a PR at the beginning, and still have a CI.

Reply:
`on: push` is simpler, but since you also need PR on master branch only, you need to do this :
```
on:
  push:
  pull_request:
    branches: [master]
```

Reply:
Done